example/settings/dev.py

```diff
@@ -74,7 +79,13 @@
     ),
     'DEFAULT_RENDERER_CLASSES': (
         'rest_framework_json_api.renderers.JSONRenderer',
-        'rest_framework.renderers.BrowsableAPIRenderer',
+
+        # If you're performance testing, you will want to use the browseable API
+        # without forms, as the forms can generate their own queries.
+        # If performance testing, enable:
+        'example.utils.BrowsableAPIRendererWithoutForms',
+        # Otherwise, to play around with the browseable API, enable:
+        # 'rest_framework.renderers.BrowsableAPIRenderer',
```

Comment:
Since `example` is what runs on the demo site, should we leave the form version on by default instead of the perf version so that people can add extra data to play with it?